src/storage/data_table.cpp

```diff
@@ -60,6 +60,8 @@ void DataTable::Scan(const common::ManagedPointer<transaction::TransactionContex
 }
 
 DataTable::SlotIterator &DataTable::SlotIterator::operator++() {
+  // TODO(Lin): We need to temporarily comment out this latch for the concurrent TPCH experiments. Should be replaced
```

Comment:
I prefer to keep the comment there because (1) it is a real TODO that we need to address at some point (2) if we're going to run TPCH benchmark, this will be a pointer to comment out the latches. I intentionally revert latch commenting because I don't want to break the correctness of the system.